### DISCUSSION TITLE
mdserver-web真的很棒！给mdserver-web点个赞

### DISCUSSION THREAD

**Discussion by ycgz2008** [midoks/mdserver-web#502] [General]
今天重新安装测试，发现mdserver-web已经可以当主力使用了，为mdserver-web点个赞！
防火墙
统计
谷歌备份
反向代理
ssl
超级好用，需要的都有了

再给mdserver-web提些建议
1、老子面板？天王老子？名字带有歧义，mdserver-web模仿的宝塔，起个 mb（美宝、明宝、满宝）、mt（美塔、明天、馒头）那个名字不比老子面板强，当然这是个人审美
2、建议swap 1.1可以升级为工具箱，增加修改DNS，时区、hosts（这些功能真的很实用，而且不用维护）
3、增加openrasp管理器（安全啊）
最后的最后，感谢mdserver-web的开发者，已经可以让我当主力使用了，稳定才是王道，不建议在进行添加其他插件，毕竟是个人开发，不是团队，在精不在多，是兴趣不是主业，该有的功能已经全部都有了，维护好现在的，不让开发mdserver-web增加精力（个人的精力有限、个人的精力有限、个人的精力有限），愉悦的开发，才能长期保持这个项目的长久
很多项目开发到最后都是越来越臃肿，并不是不好，而是对个人开发者来说，太大了，大的你维护起来都力不从心了
对现在的mdserver-web我已经很满意了
以上只是个人见解

**Comment by midoks** [comment]
感谢，已经进入稳定期了，改动不会太大。该有的，都会有。
